frontend/src/routes/(app)/report/[id]/[name]/+page.svelte

```diff
@@ -132,13 +188,31 @@
 				})}
 			</p>
 			<div class="mt-4 flex items-center text-lg">
-				<p class="mr-2 font-medium text-grey-dark">Author:</p>
-				<Icon class="mr-1 h-6 w-6" tag="svg" viewBox="0 0 24 24">
-					<path class="fill-grey" d={mdiAccountCircleOutline} />
-				</Icon>
-				<p>
-					{data.report.ownerName}
-				</p>
+				<p class="mr-2 font-medium text-grey-dark">Author{authors.length > 1 ? 's' : ''}:</p>
+				{#if data.report.editor}
+					<MultiSelect
+						bind:selected={authorsSelected}
+						options={authorOptions}
+						key={JSON.stringify}
+						liSelectedClass="!bg-primary-light ![&>svg]:fill-primary"
+						ulSelectedClass="!overflow-x-auto !w-full"
+						outerDivClass="!w-full !border-grey-light !py-1 !bg-white"
+						liActiveOptionClass="!bg-primary-light"
+					>
+						<p style="text-wrap: pretty;" slot="selected" let:option>{option.label}</p>
+					</MultiSelect>
+				{:else}
+					{#each authors as author}
+						<div
+							class="mr-1 flex w-fit items-center rounded bg-primary-light px-2 py-1 font-medium transition hover:bg-primary-mid"
+						>
+							<Icon class="mr-1 h-6 w-6" tag="svg" viewBox="0 0 24 24">
```

Comment:
Style suggest, no color or hover since not a clickable button:

```suggestion
						<div class="mr-1 flex w-fit shrink-0 items-center rounded px-2 py-1">
```